ruby-webapp-onbuild/Dockerfile

```diff
@@ -15,15 +15,18 @@ ENV SSH_AUTH_SOCK /tmp/ssh-auth
 ENV SSH_AUTH_PROXY_PORT 1234
 
 RUN mkdir -p /usr/src/app
+RUN bundle config --global without test:development
 WORKDIR /usr/src/app
 
 ONBUILD COPY Gemfile /usr/src/app/
 ONBUILD COPY Gemfile.lock /usr/src/app/
 
+ONBUILD COPY . /usr/src/app
```

Comment:
Gah, very good point.

Note to self, maybe try this:
`ONBUILD COPY *.sh /usr/src/app/`